packages/relay-compiler/bin/RelayCompilerMain.js

```diff
@@ -174,86 +186,105 @@ Root files can include:
 - A .hg/ Mercurial folder
 - A .watchmanconfig file
 
-Ensure that one such file exists in ${srcDir} or its parents.
+Ensure that one such file exists in ${src} or its parents.
     `.trim(),
     );
   }
-  if (options.verbose && options.quiet) {
+  if (config.verbose && config.quiet) {
     throw new Error("I can't be quiet and verbose at the same time");
   }
 
-  const reporter = new ConsoleReporter({
-    verbose: options.verbose,
-    quiet: options.quiet,
+  const watchman = config.watchman && (await WatchmanClient.isAvailable());
```

Comment:
hmm the previous name made it clear that this wasn't identical to `config.watchman` -  the logic has actually changed, see comment below